common/static/common/js/discussion/discussion_router.js

```diff
@@ -129,9 +128,7 @@
             DiscussionRouter.prototype.navigateToThread = function(thread_id) {
                 var thread;
                 thread = this.discussion.get(thread_id);
-                return this.navigate("" + (thread.get("commentable_id")) + "/threads/" + thread_id, {
-                    trigger: true
-                });
+                return this.showThread(thread);
```

Comment:
@bradenmacdonald The difference between `navigateToThread` and `showThread

I pushed an update that allows the URL to change when clicking on a specific thread.